src/main/scala/view/scalaFX/FXControllers/BaseAppController.scala

```diff
@@ -151,4 +173,29 @@ class BaseAppController(private val simulationPane: AnchorPane,
     selectedBunny.get.addClickedEffect()
     chartSelectionPanelController --> { _.handleBunnyClick() }
   }
+
+  private def setImageBackground(summerImage: ClimateImage, winterImage: ClimateImage, climate: Climate): Unit = climate match {
+    case Winter() => simulationPane.background = winterImage
+    case Summer() => simulationPane.background = summerImage
+  }
+  override def changeBackgroundEnvironment(factors: Factors, climate: Climate): Unit = {
+    if (factors.contains(DisturbingFactors.TOUGH_FOOD) && factors.contains(DisturbingFactors.HIGH_FOOD) && factors.contains(DisturbingFactors.LIMITED_FOOD)){
+     setImageBackground(SummerImageRareHighToughFood(), WinterImageRareHighToughFood(), climate)
+    } else if (factors.contains(DisturbingFactors.TOUGH_FOOD) && factors.contains(DisturbingFactors.HIGH_FOOD)) {
+      setImageBackground(SummerImageHighToughFood(), WinterImageHighToughFood(), climate)
+    } else if (factors.contains(DisturbingFactors.HIGH_FOOD) && factors.contains(DisturbingFactors.LIMITED_FOOD)) {
+      setImageBackground(SummerImageRareHighFood(), WinterImageRareHighFood(), climate)
+    } else if (factors.contains(DisturbingFactors.TOUGH_FOOD) && factors.contains(DisturbingFactors.LIMITED_FOOD)) {
+      setImageBackground(SummerImageRareToughFood(), WinterImageRareToughFood(), climate)
+    } else if (factors.contains(DisturbingFactors.TOUGH_FOOD)) {
+      setImageBackground(SummerImageToughFood(), WinterImageToughFood(), climate)
+    } else if (factors.contains(DisturbingFactors.HIGH_FOOD)) {
+      setImageBackground(SummerImageHighFood(), WinterImageHighFood(), climate)
+    } else if (factors.contains(DisturbingFactors.LIMITED_FOOD)) {
+      setImageBackground(SummerImageRareFood(), WinterImageRareFood(), climate)
+    } else {
+      setImageBackground(SummerImage(), WinterImage(), climate)
+    }
+
```

Comment:
Questo non va qui, avevo creato apposta la classe `ViewImages` per separare tutti questi controlli sulle imagini mostrate a display. Inoltre non è possibile scriverlo in modo più funzionale, così è veramente poco leggibile...